src/sql/azdata.proposed.d.ts

```diff
@@ -446,6 +446,30 @@ declare module 'azdata' {
 		showOnConnectionDialog?: boolean;
 	}
 
+	// Object Explorer interfaces  --------------------------------
+	export interface ObjectExplorerSession {
+		/**
+		 * Access token used in current session.
+		 */
+		token?: string | undefined;
+		/**
+		 * Expiry time of the access token.
+		 */
+		expiresOn?: string | undefined;
```

Comment:
I'd suggest using the AccountSecurityToken type here instead. This has a couple of benefits : 

1. Reduces "duplicate" property definitions
2. Keeps things more consistent whenever security tokens are being used
3. Better typing for the properties that are a part of the token (with what you have there's no way to know that the token property will always be present if a token is being used unless you read the docs and hope they're accurate)

```suggestion
		/**
		 * Authentication token for the current session.
		 */
		token?: AccountSecurityToken | undefined;
```